src/Autocomplete/assets/README.md

```diff
@@ -14,6 +14,18 @@ composer require symfony/ux-autocomplete:2.23.0
 npm add @symfony/ux-autocomplete@2.23.0
 ```
 
+To prevent your `package.json` file from being overwritten by Symfony Flex when upgrading the PHP package, you must **use at least Symfony Flex ^1.22.0 or ^2.5.0**, and configure your `composer.json` file like this:
```

Comment:
What about using the github markdown note ?

Reply:
Does it work on NPM website?

Reply:
I honestly dont know (same for packagist and other sf php repo readme)
But at least Github display info is clearer and denote this specific paragraph